src/components/memberList.js

```diff
@@ -1,9 +1,37 @@
 import Member from "./member";
-export default function MemberList({ members, onDeleteMember }) {
-  const membersList = members.map(member =>
-    <Member key={member.id} id={member.id} name={member.name} onDelete={onDeleteMember}></Member>)
-    ;
+import DeleteMembers from "./deleteMembers";
+import { useState } from "react";
+
+export default function MemberList({
+  members,
+  onDeleteMember,
```

Comment:
We can reuse the deleteMembers to implement deleting a single member right? So we don't have to provide 2 props of deleting single/multiple members